pino.js

```diff
@@ -338,6 +343,15 @@ function pino (opts, stream) {
     }
   }
 
+  const baseLog = typeof iopts.baseLog === 'object' ? iopts.baseLog : defaultBaseLog
```

Comment:
let's not call it baseLog

how about just calling it either `baseKeys` or `base`

just to define it further

* defaults to having pid and hostname
* may only be an object
* these can be removed if set to `false` e.g. `{base: { pid: false }}`
* can also have other preset values which will be added into `_baseLog` e.g. `{base: { container: 'my-container-name' }}`
* child loggers inherit the base (I think this happens anyway)